src/studio/src/designer/backend/Services/Implementation/LanguagesService.cs

```diff
@@ -27,7 +27,7 @@ public IList<string> GetLanguages(string org, string repo, string developer)
         {
             var altinnGitRepository = _altinnGitRepositoryFactory.GetAltinnAppGitRepository(org, repo, developer);
 
-            IList<string> languages = new List<string>();
+            List<string> languages = new List<string>();
```

Comment:
Should return the model ie. Dictionary<string, string>

Reply:
For languages also? Thought that was just a list of the available languages? 🤔

Reply:
Sorry, wrong comment in wrong place - just ignore:)